rollup.config.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright (c) 2018 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import filesize from 'rollup-plugin-filesize';
+import {terser} from 'rollup-plugin-terser';
+import resolve from 'rollup-plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+
+export default {
+  input: 'my-element.js',
+  output: {
+    file: 'my-element.bundled.js',
```

Comment:
Is this cleaned up, published to npm, or checked in?

Reply:
This is just copied from the TS project. Rollup is used for the checksize script (the bundle is cleanup up afterwards) and for the generated docs (that version gets checked in).

Reply:
Yeah, it's deleted in the checksize script. Can we add it to .gitignore in case that does wrong somehow?

Reply:
Done. Eeek--thanks, realized the .gitignore was copied from the TS version and quite wrong 🐐